compiler/crates/relay-config/src/typegen_config.rs

```diff
@@ -114,6 +114,10 @@ pub struct TypegenConfig {
     #[serde(default)]
     pub typescript_exclude_undefined_from_nullable_union: bool,
 
+    // todo: should this go here or in a feature flag? Above flag is discussed here: https://github.com/facebook/relay/pull/4380
```

Comment:
Feature flag is probably slightly preferable if it's feasible to thread it through without too much trouble. If it's very noisy to thread it through, this is fine.

Reply:
Changed to feature flag!